charts/aws-efs-csi-driver/values.yaml

```diff
@@ -136,6 +153,13 @@ node:
     ## Enable if EKS IAM for SA is used
     #  eks.amazonaws.com/role-arn: arn:aws:iam::111122223333:role/efs-csi-role
   healthPort: 9809
+  # securityContext on the node pod
+  securityContext:
+    # The node pod must be run as root to bind to the registration/driver sockets
+    runAsNonRoot: false
```

Comment:
Bit confused here

From the comment it's saying that `The node pod must be run as root to bind to the registration/driver sockets`

but then why we set `runAsNonRoot: false` ?

Reply:
`runAsNonRoot: false` which means `root: true`. We are explicitly declaring important security configurations.